src/Wrapper.js

```diff
@@ -346,13 +344,9 @@ class Wrapper extends React.PureComponent {
 
     if (instanceId === 'home') {
       return (
-        <Home
-          apps={apps}
-          connected={connected}
-          dao={locator.dao}
-          onMessage={this.handleAppMessage}
-          onOpenApp={this.openApp}
-        />
+        <AppInternal>
```

Comment:
Wonder if it would make sense to wrap Home within its own file? What do you think?

Reply:
You mean in a different way than it is now? https://github.com/aragon/aragon/pull/934/files#diff-cdace88e2d25deba8b1d9ed1f6560baf

Reply:
Just wondering if it should be:
```JavaScript
export default React.memo(props => <AppInternal><Home {..props} /></AppInternal>)
```

To avoid more code in `Wrapper`, no other reason.

Reply:
Oh I see, yes good idea! We are probably going to wrap every app in `OrgView` with `Root`, so maybe we can do this + move all the `AppInternal` inside the apps at the same time :+1: